src/main/java/com/team/buddyya/student/exception/StudentExceptionType.java

```diff
@@ -7,6 +7,7 @@ public enum StudentExceptionType implements BaseExceptionType {
 
     STUDENT_NOT_FOUND(200, HttpStatus.NOT_FOUND, "해당 학생을 찾지 못하였습니다."),
     UNIVERSITY_NOT_FOUND(200, HttpStatus.NOT_FOUND, "해당 대학을 찾지 못하였습니다."),
+    Major_NOT_FOUND(200, HttpStatus.NOT_FOUND, "해당 전공을 찾지 못하였습니다."),
```

Comment:
P2: 다른 enum과 마찬가지로
```suggestion
    MAJOR_NOT_FOUND(200, HttpStatus.NOT_FOUND, "해당 전공을 찾지 못하였습니다."),
```
통일하는 건 어떤가요?